imagersite/imagersite/settings.py

```diff
@@ -124,8 +157,13 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.9/howto/static-files/
 
+# STATIC_ROOT = os.path.join(BASE_DIR, 'static')
 STATIC_URL = '/static/'
+STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+STATICFILES_DIRS = [
+    os.path.join(BASE_DIR, 'static'),
```

Comment:
`STATIC_ROOT` and `STATICFILES_DIRS` can never point to the same place.

`STATIC_ROOT` signifies the location into which all files found by `manage.py collectstatic` will be collected. This is the directory that you would point nginx at when you tell it how to serve resources whose URIs begin with `/static/`.

`STATICFILES_DIRS` is a list of additional directories from which to _source_ files with `manage.py collectstatic`. By default, `collectstatic` will walk through every app in your project; for each app that contains a `static/` directory, it will pull the contents of that directory out into the `STATIC_ROOT` directory. Additionally, if `STATICFILES_DIRS` is specified, `collectstatic` will also walk through the locations listed in `STATICFILES_DIRS` and pull any static files found there out into the `STATIC_ROOT` directory.

What you've done here, by setting `STATIC_ROOT = STATICFILES_DIRS = os.path.join(BASE_DIR, 'static')`, is tell `manage.py collectstatic` to look for static files in `/path/to/imagersite/static/`, and then put those files into `/path/to/imagersite/static/` -- 'for everything you find in this directory, put them into that directory in which they already live.'